force-app/main/default/classes/JoobleCallout.cls

```diff
@@ -0,0 +1,191 @@
+public with sharing class JoobleCallout {
+    public static Integer resultsPerPage = 100;
+    public static List<String> keywordList = new List<String>();
+    public static Set<Id> newAppIds = new Set<Id>();
+    public static Integer pageNumber = 1;
+    public static Integer maxPage = 1;
+    public static Boolean maxPageSet = false;
+    public static List<JoobleListingWrapper.Job> salesforceListings = new List<JoobleListingWrapper.Job>();
+    public static Set<String> matchingIds = new Set<String>();
+    public static List<Application__c> newApplications = new List<Application__c>();
+
+    @AuraEnabled
+    public static List<Application__c> getJoobleListings(String keywords, String location, String fromSearchDate) {
+        System.debug('keywords: ' + keywords);
+        System.debug('location: ' + location);
+        System.debug('fromSearchDate: ' + fromSearchDate);
+        // Capture all keywords in a list to check against listing title later
+        keywordList = keywords.split(' ');
+        // Update keywords to just 'Salesforce' if keywords contains 'Salesforce' to minimize API calls
+        // Later, keywordList will be used to check if all keywords are present in the title
+        if (keywordList.contains('Salesforce')) {
+            keywords = 'Salesforce';
+        }
+        System.debug('keywordList: ' + keywordList);
+        System.debug('keywords: ' + keywords);
+
+        // The while loop handles multiple pages of results which requires repeat callouts
+        while (pageNumber <= maxPage) {
```

Comment:
I really like your implementation of fetching multiple pages of data from the JoobleAPI using a while loop! As you mentioned in your presentation, limits could become an issue though if there are large quantities of results pages. Maybe consider implementing a mechanism to throttle or batch the callouts?